src/cast_core/src/Loader.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include <algorithm>
 #include <memory>
+#include <locale>
```

Comment:
Do we need this extra include?

Reply:
This was from Ger's change. Let me try removing it and see if it builds.

Reply:
Nope -- not needed.